apps/catalogue/components/ResourceCard.vue

```diff
@@ -58,13 +71,24 @@ const iconStarClasses = computed(() => {
         </span>
       </div>
       <div class="flex">
-        <!--
-        <IconButton
-          icon="star"
-          :class="iconStarClasses"
-          class="text-blue-500 xl:justify-end"
-        />
-        -->
+        <label
+          :for="`${resource.id}-shopping-cart-input`"
+          class="xl:flex xl:justify-end px-2 py-1 rounded-3px cursor-pointer text-blue-500 hover:text-blue-800 focus:text-blue-800"
+          :class="{
+            'items-baseline xl:items-center mt-0.5 xl:mt-0': !compact,
+            'bg-blue-500 text-white hover:text-white': isInShoppingCart,
+          }"
+        >
+          <BaseIcon name="shopping-cart-add" :width="21" />
```

Comment:
I would like another icon but I also want to complete the story soon so I don't mind to do that later